scalding-core/src/test/scala/com/twitter/scalding/StringUtilityTest.scala

```diff
@@ -0,0 +1,95 @@
+package com.twitter.scalding
+
+import org.scalatest.{ Matchers, WordSpec }
+
+class StringUtilityTest extends WordSpec with Matchers {
```

Comment:
how about a test / property test that says "fast split should behave the same as String.split?